contracts/aggchains/AggchainECDSAMultisig.sol

```diff
@@ -23,7 +24,7 @@ contract AggchainECDSAMultisig is AggchainBase {
     bytes2 public constant AGGCHAIN_TYPE = 0x0000;
 
     /// @notice Aggchain version
-    string public constant AGGCHAIN_ECDSA_MULTISIG_VERSION = "v1.0.0";
+    string public constant AGGCHAIN_ECDSA_MULTISIG_VERSION = "v1.0.1";
```

Comment:
unsure if this makes sense. breaks a lot of stuff. but since the contract is changing we need versioning? using semver, +0.0.1 since its not breaking